change/@internal-react-composites-4beccd45-629a-4c8a-9905-daf1e36ccee0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
nit: type should be `none`. Playwright is a `devDependency` that doesn't affect the final package in any way.